FlexMeter.c

```diff
@@ -0,0 +1,226 @@
+/*
+htop - FlexMeter.c
+(C) 2024 Stoyan Bogdanov
+Released under the GNU GPLv2+, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "config.h"
+#include "FlexMeter.h"
+#include <time.h>
+#include <sys/time.h>
+#include <dirent.h>
+#include "CRT.h"
+#include "Object.h"
+
+#include <unistd.h>
+#include <sys/types.h>
+#include <pwd.h>
+
+#define FLEX_CFG_FOLDER ".config/htop/FlexMeter"
+
+typedef struct {
+   char* name;
+   char* command;
+   char* type;
+   char* caption;
+   char* uiName;
+}_flex_meter;
+
+_flex_meter meter_list[30];
+
+int meter_list_idx=0;
+static int meters_count=0;
+
+static const int DateMeter_attributes[] = {
+   FLEX
+};
+
+MeterClass * FlexMeter_class = NULL;
+
+int check_for_meters(void);
+
+static int parse_input(char * line)
+{
+   for (int i = strlen(line);i>0;i--)
+   {
+      if ((line[i]=='\r') || (line[i]=='\n'))
+      {
+         line[i]='\0';
+      }
+   }
+
+   if (!strncmp(line,"name=",5))
```

Comment:
Please use `String_startsWith` when possible. It saves your time manually updating the string length.

By the way, if we are forbidding whitespace between `name` and equal sign, better state that explicitly in the documentation.

Reply:
> Please use `String_startsWith` when possible. It saves your time manually updating the string length.

Replacing it with String_startsWith is causing - `Segmentation fault`.

> By the way, if we are forbidding whitespace between `name` and equal sign, better state that explicitly in the documentation.

Is man page enough ?

Reply:
> Is man page enough ?

As I have comments about the current format of the template file, I think we can think of updating the man page later. Yes, the format is supposed to be documented, but we can think of documentation later, until the code is stable enough.